backend/api/office_hours/event.py

```diff
@@ -1,287 +1,66 @@
-"""OH Event API
+"""Office Hours API
 
-This API is used to access OH Event data."""
-
-from datetime import date, datetime, timedelta
-from fastapi import APIRouter, Depends, HTTPException
-from sqlalchemy import Date, String
-
-from ...models.office_hours.event_status import (
-    OfficeHoursEventStatus,
-    StaffHelpingStatus,
-    StudentOfficeHoursEventStatus,
-    StudentQueuedTicketStatus,
-)
-
-from ...models.office_hours.ticket_details import OfficeHoursTicketDetails
-from ...models.coworking.time_range import TimeRange
-from ...models.office_hours.event import (
-    OfficeHoursEvent,
-    OfficeHoursEventDraft,
-    OfficeHoursEventPartial,
-    OfficeHoursEventRecurringDraft,
-)
-from ...models.office_hours.event_details import OfficeHoursEventDetails
-from ...services.office_hours.event import OfficeHoursEventService
+APIs handling office hours.
+"""
 
+from fastapi import APIRouter, Depends
 from ..authentication import registered_user
-from ...models import User
+from ...services.office_hours.event import OfficeHourEventService
+from ...models.user import User
 
+from ...models.academics.my_courses import (
+    OfficeHourQueueOverview,
+    OfficeHourEventRoleOverview,
+    OfficeHourGetHelpOverview,
+)
 
-__authors__ = ["Sadie Amato", "Madelyn Andrews", "Bailey DeSouza", "Meghan Sun"]
+__authors__ = ["Ajay Gandecha"]
```

Comment:
We should leave attribution here even though the rewrite is significant; not starting from a position of scratch was thanks to the initial effort of the original authors. Additionally, they'll be in the git log with blame, etc.

Reply:
@KrisJordan sorry! I think this was caused because I deleted the original file and created a new one with the same name, causing this overwrite. Will revert 🫡